src/test/java/lotto/controller/LottoControllerTest.java

```diff
@@ -0,0 +1,66 @@
+package lotto.controller;
+
+import lotto.domain.LottoResult;
+import lotto.domain.Prize;
+import lotto.domain.machine.AutoLottoMachine;
+import lotto.domain.ticket.LottoTicket;
+import lotto.domain.ticket.LottoTickets;
+import lotto.domain.ticket.WinningLottoTicket;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class LottoControllerTest {
+    LottoController lottoController;
+
+    @BeforeEach
+    void setup(){
+        InputView inputView = new InputView(new Scanner(System.in));
+        lottoController = new LottoController(inputView, new OutputView(), new AutoLottoMachine());
+    }
+
+    @DisplayName("로또 티켓 구매 메소드 확인")
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3,4,5,6,7,8,10})
```

Comment:
```suggestion
    @ValueSource(ints = {1, 2, 3, 4, 5, 6, 7, 8, 10})
```

추가적으로 테스트를 돌리면 output에 뷰관련하여 계속 메세지가 찍힙니다. 
일례로 컨트롤러의 buyLottoTicket이라는 메서드를 보시면 뷰를 그려주는 역할도 담당하고있습니다. 해당 메서드는 정말 그 기능만 하고 외부에서 뷰를 호출해주게되면 테스트에서도 콘솔에 메시지가 찍히지않게되고 메서드가 하나의 역할만 담당할수도있을거같아요!

Reply:
동의합니다! 수정하였습니다 😄